src/test/applytx_tests.cpp

```diff
@@ -0,0 +1,176 @@
+#include <chainparams.h>
```

Comment:
@uzyn @monstrobishi Do we need add defichain copyright information for the new files added by us?